packages/adapter-cloudflare/package.json

```diff
@@ -33,6 +33,7 @@
 	},
 	"dependencies": {
 		"@cloudflare/workers-types": "^4.20231121.0",
+		"wrangler": "^3.24.0",
```

Comment:
Will users expect to install `wrangler` and control what version of `wrangler` is used? I'm wondering if it's better here or in `peerDependencies`. You probably know better than I do though

Reply:
Yes, when developing their app they will use `wrangler` anyways so I think that it makes sense to have it as a peer dependency, I've moved it 👍

But with this change, since `wrangler` might actually not be installed, I guess I might need to change the wrangler import for `getBindingsProxy` to be dynamic and try-catched so that we can check if it actually succeeds

What do you think?

Reply:
Yes, putting it in a try-catch so we can print a nice error message when it is not present makes sense to me

Reply:
Can you regenerate the pnpm lock file as well?

Reply:
I did re-run `pnpm i` but it didn't generate any different in the pnpm lock file, I guess it should? 😕